index.bs

```diff
@@ -294,6 +294,7 @@ Parts of the web platform evolve independently.
 Issues that are present with a certain web technology now may be fixed in a subsequent iteration.
 Duplicating these issues makes fixing them more difficult.
 By adhering to this principle we can make sure overall platform quality improves over time.
+
```

Comment:
```suggestion
```

Reply:
Vertical whitespace is useful.  Why would you remove the space around the h2?

(A more serious problem here is the lack of content between the h2 and the first h3.  That's very bad form.